WORKSPACE

```diff
@@ -5,9 +5,14 @@ load("//builders/bazel:deps.bzl", "python_deps")
 
 python_deps("//builders/bazel")
 
-local_repository(
+http_archive(
     name = "google_privacysandbox_servers_common",
-    path = "data-plane-shared-libraries",
```

Comment:
What about this?

Reply:
The demo repo does this change as a patch

Reply:
This is ok as Dom has a patch to make sure we use the local version.